app/models/katello/candlepin/content.rb

```diff
@@ -144,7 +144,11 @@ def katello_content_type
       end
 
       def unprotected?
-        kickstart? ? true : false
+        (kickstart? || file?) ? true : false
```

Comment:
Do we need the ternary operator here? Since `(kickstart? || file?)` is a boolean expression already, can the function not simply return the result of that evaluation?